src/main/resources/main.js

```diff
@@ -0,0 +1,67 @@
+var projectLib = require('/lib/xp/project');
+var contextLib = require('/lib/xp/context');
+var clusterLib = require('/lib/xp/cluster');
+
+var projectData = {
+    id: 'sample-blog',
```

Comment:
wouldn't in be better to call superhero-blog, so it is clear what creates this repo